### PR TITLE
release changelog

### PR DESCRIPTION
**Pull request by capmayer**
Release already done hehe

### CODE REVIEW

**Comment by nicoddemus** [comment]
Well done! That was all there is to it. 😁 

(Ideally you should wait for an approval before pushing a tag, because pushing the tag releases it to [PyPI](https://pypi.org/project/barril/))

**Review by nicoddemus — APPROVED** [review]

**Comment by capmayer** [comment]
> Well done! That was all there is to it. grin
> 
> (Ideally you should wait for an approval before pushing a tag, because pushing the tag releases it to [PyPI](https://pypi.org/project/barril/))

Yeah, I was reading each step and doing it, then the last step is "merge the PR" that I haven't created because there was no mention to it before hehe (the build check is done anyway just pushing a new branch)

**Comment by nicoddemus** [comment]
> Yeah, I was reading each step and doing it, then the last step is "merge the PR" that I haven't created, because there was no mention to it before hehe

Feel free to open a PR adjusting that. 👍

**Pull request merged**
